containers/ecr-viewer/sql/core.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS ecr_data (
+  eICR_ID VARCHAR(200) NOT NULL,
+  data_source VARCHAR(2) NOT NULL, -- S3 or DB
+  data_link VARCHAR(500), -- Link to the data
+  patient_name_first VARCHAR(100) NOT NULL,
+  patient_name_last VARCHAR(100) NOT NULL,
+  patient_birth_date DATE NOT NULL,
+  report_date DATE,
+  PRIMARY KEY (eICR_ID)
+);
+
+CREATE TABLE IF NOT EXISTS ecr_rr_conditions (
+    uuid UUID PRIMARY KEY,
+    eICR_ID UUID REFERENCES ecr_data(eICR_ID),
+    condition VARCHAR(255)
+    rule_summary TEXT[]
```

Comment:
I think we are missing a comma at the end of condition
```suggestion
    condition VARCHAR(255),
    rule_summary TEXT[]
```